app/views/candidate_interface/course_choices/options_for_course.html.erb

```diff
@@ -8,8 +8,8 @@
 
       <%= f.govuk_radio_buttons_fieldset :code, legend: { size: 'xl', text: t('page_titles.which_course') } do %>
         <div class="govuk-!-margin-top-6">
-          <% @pick_course.available_courses.map do |course| %>
-            <%= f.govuk_radio_button :code, course.code, label: { text: "#{course.name} (#{course.code})" } %>
+          <% @pick_course.available_courses.each_with_index do |course, i| %>
```

Comment:
Could `@pick_course.available_courses` ever be empty?

Reply:
Yes, I think it can. This probably needs thinking about, but maybe not as part of this card